src/utils.ts

```diff
@@ -1,6 +1,9 @@
+import along from "@turf/along";
+import distance from "@turf/distance";
+import { lineString, point } from "@turf/helpers";
```

Comment:
Looks like the only thing used from these imports is:
```suggestion
import { point } from "@turf/helpers";
```